browse/templates/home/special-message.html

```diff
@@ -19,3 +21,15 @@ <h2>TeX Live 2023 Upgrade</h2>
   <p><a href="https://info.arxiv.org/hiring/">View open roles</a></p>
 </div>
 {%- endif -%}
+
+{%- if config['BROWSE_SPECIAL_MESSAGE_ENABLED'] and rd_int >= 202310010100 and rd_int <= 202312300100 -%}
+<div class="message-special dark column">
+  <span class="label">arXiv is moving!</span>
+  <p>Experience the future with our cloud-powered app today!</p>
```

Comment:
```suggestion
  <p>arXiv is preparing for our next 30 years by transitioning from our current aging infrastructure to stable cloud-hosted services. </p>
```

Reply:
I'd suggest "arxiv is preparing for our next 30 years by transitioning to cloud-hosted services"

Reply:
Let's not put this out there until we have a landing page that explains the plan more fully. I don't want to introduce the whole idea of the cloud transition with a one sentence message.